tests/sqllogictests/suites/base/06_show/06_0001_show_create_table

```diff
@@ -28,6 +28,14 @@ SHOW CREATE TABLE `test`.`b`
 ----
 b CREATE TABLE `b` (   `a` BIGINT,   `b` INT NULL DEFAULT NULL,   `c` VARCHAR,   `d` SMALLINT UNSIGNED NULL ) ENGINE=NULL COMMENT='test b'
 
+statement ok
+create view test.v_b as select * from `test`.`b`
+
+query TT
+show create table test.v_b
+----
+v_b CREATE TABLE `v_b` (   `a` BIGINT,   `b` INT NULL,   `c` VARCHAR,   `d` SMALLINT UNSIGNED NULL ) ENGINE=VIEW QUERY='SELECT * FROM `test`.`b`'
```

Comment:
better to display it as original SQL
```
create view test.v_b as select * from `test`.`b`
```

Reply:
> better to display it as original SQL
> 
> ```
> create view test.v_b as select * from `test`.`b`
> ```

Yes. If want to known the fileds just use `desc view_name`

Reply:
Fixed.